src/assignmentOverride.test.js

```diff
@@ -74,34 +80,33 @@ describe('AssignmentOverride', () => {
 
   describe('#persistAssignment()', () => {
     it('creates an assignment on the test track server', () => {
-      testContext.override.persistAssignment();
-
-      expect(client.post).toHaveBeenCalledTimes(1);
-      expect(client.post).toHaveBeenCalledWith(
-        '/v1/assignment_override',
-        {
-          visitor_id: 'visitorId',
-          split_name: 'jabba',
-          variant: 'cgi',
-          context: 'spec',
-          mixpanel_result: 'success'
-        },
-        {
-          auth: {
-            username: 'the_username',
-            password: 'the_password'
-          }
-        }
-      );
+      return testContext.override.persistAssignment().then(() => {
+        expect(mockClient.history.post.length).toBe(1);
+        expect(mockClient.history.post[0].url).toEqual(expect.stringContaining('/v1/assignment_override'));
```

Comment:
nit: Why are we using a contains matcher here instead of matching on the exact string?

Reply:
For brevity, the exact string contains the mocked baseUrl. I can update it to the full url for clarity, otherwise it seemed irrelevant for these tests.

Reply:
Fine by me. I think this is actually preferable bc now we can change our underlying defaults without having to update all of our tests. Although that may be a bug rather than a feature depending on how you look at it 🤷‍♂.

Reply:
I think that this is great and desirable. We don't care about the host portion in these tests. That portion is variable across consumers of this client, but the paths are not, those are the important bits.